webservices/resources/reports.py

```diff
@@ -139,7 +142,8 @@ def build_query(self, committee_type=None, **kwargs):
             query = query.filter(reports_class.is_amended == kwargs['is_amended'])
 
         query = filters.filter_range(query, kwargs, get_range_filters())
-
+        #currently doesn't work, whyyyyy
+        #query = filters.filter_match(query, kwargs, get_match_filters())
```

Comment:
That's definitely curious, it doesn't work at all?  Is this why `filer_type` appears above?